framework/include/problems/FEProblemBase.h

```diff
@@ -1998,6 +1998,11 @@ class FEProblemBase : public SubProblem, public Restartable
    */
   unsigned int nlSysNum(const NonlinearSystemName & nl_sys_name) const;
 
+  /**
+   * Skip further residual evaluations and fail the next nonlinear convergence check
+   */
+  bool failNextNonlinearConvergenceCheck() { return _fail_next_nonlinear_convergence_check; }
```

Comment:
Looks like you missed my suggested const addition

Reply:
Yeah, I did. Sorry!